src/ert/gui/model/snapshot.py

```diff
@@ -115,11 +114,12 @@ def prerender(
                 snapshot.data()[ids.REALS].keys(), key=int
             )
             metadata[SORTED_JOB_IDS] = {}
-            for real_id, real in snapshot.data()[ids.REALS].items():
+            for real_id in snapshot.reals.keys():
                 metadata[SORTED_JOB_IDS][real_id] = {}
-                for step_id, step in real[ids.STEPS].items():
+                for step_id in snapshot.steps(real_id).keys():
```

Comment:
I know refactoring will tackle this, just it hurts looking at redundant for loops in python :laughing:

Reply:
yes, it is : )